app/environments/brassbirmingham/brassbirmingham/envs/__init__.py

```diff
@@ -0,0 +1 @@
+from brassbirmingham.envs.brassbirmingham import BrassBirminghamEnv
```

Comment:
nit: include newline at EOF

Reply:
For what purpose master chief?

Reply:
So all our files are consistent and so Github UI doesn't show that symbol indicating there's no newline at EOF

Reply:
Covered by black